src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -1,19 +1,45 @@
 (ns puppetlabs.puppetdb.http.command
   (:require [puppetlabs.puppetdb.command.constants :refer [command-names]]
             [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [clojure.string :as str]
             [clojure.tools.logging :as log]
             [puppetlabs.puppetdb.command :as command]
             [puppetlabs.puppetdb.http :as http]
+            [puppetlabs.puppetdb.cheshire :as json]
             [net.cgrand.moustache :as moustache]
             [puppetlabs.puppetdb.middleware :as mid]
             [compojure.core :as compojure]))
 
+(def min-supported-commands
+  {"replace catalog" 6
+   "replace facts" 4
+   "store report" 5
+   "deactivate node" 3})
```

Comment:
Not sure I follow -- want to give an example? It sounds like you're suggesting this

```
{"replace catalog" [3 4 5 6]
 "replace facts" [4 5 6]
 ...}
```

I don't like that because it'll require an update every time we create a new command. The only important information here is the minimum supported.